drivers/ds3234/include/ds3234_params.h

```diff
@@ -47,15 +47,15 @@ extern "C" {
     { \
         .spi = DS3234_PARAM_SPI, \
         .cs = DS3234_PARAM_CS, \
-        .clk = DS3234_PARAM_CLK, \
+        .freq = DS3234_PARAM_CLK, \
     }
 #endif
 /**@}*/
 
 /**
  * @brief   Configure DS3234 devices
  */
-static const ds3234_params_t ds3234_params[] =
+static ds3234_params_t ds3234_params[] =
```

Comment:
Hm, too bad, this prevents the parameter set from being stored in ROM instead of RAM.

Reply:
For this one I didn't have a device descriptor where to store the spi clock configuration so I fell back on a caching function, I'm not sure that's an improvement...